<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="txtDurum">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHesapla">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>hesapla</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTeizle">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>temizle</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ad</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>bol√ºm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>vize notu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>final notu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>durum</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="edtAd"/>
    </item>
    <item>
     <widget class="QLineEdit" name="edtBolum"/>
    </item>
    <item>
     <widget class="QLineEdit" name="edtVize"/>
    </item>
    <item>
     <widget class="QLineEdit" name="edtFinal"/>
    </item>
    <item>
     <widget class="QLineEdit" name="edtDurum"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnTeizle</sender>
   <signal>clicked(bool)</signal>
   <receiver>txtDurum</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
